themes/finna2/js/finna-feed-element.js

```diff
@@ -177,11 +177,12 @@ class FinnaFeedElement extends HTMLElement {
           holder.querySelectorAll('.carousel-slide-more.carousel-show').forEach(el => {
             el.classList.remove('hidden');
           });
+          var feedItemWidth = holder.querySelector('li.feed-item-holder').offsetWidth;
           holder.querySelectorAll('.carousel-hover-title').forEach(el => {
-            el.style.width = '90%';
-            var element = el.querySelector('p');
-            if (element) {
-              element.style.paddingRight = '10px';
+            if (feedItemWidth > 200) {
```

Comment:
Olisi parasta, että sulkupainiketta ei tarvitsisi ronkkia js:llä. Sulkupainikkeen sijainnin pitäisi pystyä määritellämään pelkästään css-tyyleillä. Ei suoraa liitty tähän pr:ään. mutta jäin vielä ihmettelmään, miksi sulkupainikkeen ympärille on asetettu p-elementti, parempi olisi käyttää span- tai div-elementtiä.